src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1325,7 +1325,7 @@ function ($a) {
                                                         return $middleware;
                                                     }
                                                     if (1 < \count($middleware)) {
-                                                        throw new \InvalidArgumentException(sprintf('Invalid middleware at path "framework.messenger": a map with a single factory id as key and its arguments as value was expected, %s given.', json_encode($middleware)));
+                                                        throw new \InvalidArgumentException(sprintf('Invalid middleware at path "framework.messenger": a map with a single factory id as key and its arguments as value was expected, "%s" given.', json_encode($middleware)));
```

Comment:
That's a false positive as the value is JSON encoded. We can do something like this instead: `as expected, '.json_encode($middleware).' given.`

Reply:
Oh sorry, fixed